netbox/extras/api/serializers.py

```diff
@@ -179,6 +179,15 @@ class Meta:
             'choices_count', 'created', 'last_updated',
         ]
 
+    def validate_extra_choices(self, value):
```

Comment:
While this approach provides validation, it doesn't allow for easy introspection. We should declare `extra_choices` as a [ListField](https://www.django-rest-framework.org/api-guide/fields/#listfield) on the serializer, with a child ListField which requires exactly two items.